src/remediations/controller.fifi.js

```diff
@@ -26,7 +26,18 @@ exports.connection_status = errors.async(async function (req, res) {
         throw new errors.Forbidden();
     }
 
-    const status = await fifi.getConnectionStatus(remediation, req.identity.account_number, req.entitlements.smart_management);
+    const rhcEnabled = await fifi.checkRhcEnabled();
+
+    if (!rhcEnabled && !req.entitlements.smart_management) {
```

Comment:
Is `&& !req.entitlements.smart_management` there because you can only enable remediations in RHC if you have smart management?

Reply:
Well either way I think the implication is that if you purchase smart management we may run stuff on your hosts using RHC regardless of whether you have that option enabled in the RHC dashboard.

Reply:
oh!  @jharting so the reason I added that to the check is to fail if rhc is not enabled AND the customer doesn't have the smart_management entitlement, I was under the impression that if you didn't have rhcEnabled on the dashboard and didn't have the entitlement you wouldn't be able to run any playbooks  (either through RHC or Satellite)

Reply:
Have you considered using the new DISABLED state instead? I.e. regardless of whether I have smart management or not, if I have rhc-connected hosts with RHC disabled, I see x hosts in the rhc-connected-disabled bucket in preflight. I think that ways users are more likely to find the switch they need to flip than if all they get is a 403.

Reply:
I agree!  I like this alot better than sending a 403.  The newly pushed changes should remedy this!